rsts/deployment/gcp/manual.rst

```diff
@@ -480,7 +479,13 @@ To restrict domains, update Helm values again. By default, Flyte creates three d
                   value: "3000Mi"
                 - defaultIamRole:
                   value: "gsa-development@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com"
```

Comment:
value: "gsa-development@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com"
to 
 value: "gsa-staging@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com"